specs/spec-1-kudos.md

```diff
@@ -1,107 +1,269 @@
----
-#: 1
-Status: Review
-Category: NEAR Social
-Created: 2023-04-05
-Replaces: --
-Requires: --
----
+<!-- Objective. In a few sentences, describe the key system objectives. Tweet length executive summary. -->
+# RFP 2: Kudos Widget
+Kudos is a widget to build a reputation. The widget will allow any verified human to nominate, recieve, and interact with Kudos on Near Social.
+
+# Metadata
+
+<!-- Common status: Inception, Draft, Community consultation, Locked: RFP ongoing, Locked: RFP awarded, Obsolete -->
+## Status
+| Key       | Value                 |
+|-----------|-----------------------|
+| **#**    | 1                     |
+| **Status**  | Review              |
+| **Category** | NEAR Social        |
+| **Replaces** | --                 |
+| **Requires** | --                 |
+
```

Comment:
this doesn't follow the template. 
The "metadata" is a special section in markdown, must be at the top, not a table and not indicated with the header. Such document metadata have special meaning and are rendered correctly.